js/event.js

```diff
@@ -321,7 +321,7 @@ export const addEventUPSubmit = () => {
                     }
                 }
                 if(validationPattern && validationPattern === 'year') {
-                    if(!/[19|20]{2}[0-9]{2}/.test(element.value)) {
+                    if(!/(19|20)[0-9]{2}/.test(element.value)) {
```

Comment:
Suggest adding ^ and $ to indicate start & end input has pattern matching.
Nice use of () for group matching

Reply:
I agree it adds an extra layer of validation so I added it